modmesh/gui/naca.py

```diff
@@ -35,8 +35,7 @@
 from modmesh import view
 
 
-def calc_naca4_points(number, npoint, open_trailing_edge=False,
-                      cosine_spacing=False):
+def calc_naca4_points(number, npoint, open_trailing_edge=False, cosine_spacing=False):
```

Comment:
Why changing the format of this line?

Reply:
Because there is an error `./modmesh/gui/naca.py:38:80: E501 line too long (86 > 79 characters)` when I run `make flake8`, I change it for fix it. But it actually did not fix and turn out the other error `./modmesh/gui/naca.py:38:64: W291 trailing whitespace`. 

I will figure out how to deal with linting and formatting Python in vscode.

Reply:
The code after your change uses 86 characters:

```python
def calc_naca4_points(number, npoint, open_trailing_edge=False, cosine_spacing=False):
```

It seems the code before your change is OK.